src/Components/MyBooks/MyBooks.tsx

```diff
@@ -66,6 +76,17 @@ const MyBooks = ( { currentUser }: { currentUser: User | any}) => {
   }
   }, [data])
 
+  const deleteSelectedBook = (id: number) => {
+    let currentUser = JSON.parse(localStorage.currentUser)
```

Comment:
Nice!!! This is what I refactored on pretty much all the dashboard components. A super smart was of going about it